src/test/java/org/fcrepo/vocabulary/AbstractVocabularyTest.java

```diff
@@ -47,6 +50,10 @@
 
     public abstract Class vocabulary();
 
+    static {
+        setDefaultHttpClient(create().setRedirectStrategy(new LaxRedirectStrategy()).build());
```

Comment:
ok, but this is just for the test suite, and all that really matters for that is to pull a current version of the ontology and compare it to the fields defined in these classes.